src/algebra/group/to_additive.lean

```diff
@@ -551,3 +552,35 @@ attribute [to_additive empty] empty
 attribute [to_additive pempty] pempty
 attribute [to_additive punit] punit
 attribute [to_additive unit] unit
+
+section linter
+
+open tactic expr
+
+/-- A linter that checks that multiplicative and additive lemmas have both doc strings if one of
+them has one -/
+@[linter] meta def linter.to_additive_doc : linter :=
+{ test := (λ d, do
+    let mul_name := d.to_name,
+    dict ← to_additive.aux_attr.get_cache,
+    match dict.find mul_name with
+    | some add_name := do
+      mul_doc <- doc_string mul_name >> return tt <|> return ff,
+      add_doc <- doc_string add_name >> return tt <|> return ff,
+      match mul_doc, add_doc with
+      | tt, ff := return $ some $ "declaration has a docstring, but its additive version `" ++
+         add_name.to_string ++ "` does not. You might want to pass a string argument to " ++
+         "`to_additive`."
+      | ff, tt := return $ some $ "declaration has no docstring, but its additive version `" ++
```

Comment:
```suggestion
      | ff, tt := return $ some $ "declaration has no docstring, but its multiplicative version `" ++
```
and you should need to change which lemma name you print out as well.

Reply:
I don't think so, I always phrase the original version first, because that's how the linter prints it.